beancount_import/source/ofx.py

```diff
@@ -785,6 +786,10 @@ def get_security(unique_id: str) -> Optional[str]:
                 return None
             sec = securities_map[unique_id]
             ticker = sec.ticker
+            # Treasury bill and bond start with 912
+            if ticker.startswith("912"):
```

Comment:
Is the ticker of a bond always the same as its CUSIP?

Reply:
It is at least for Fidelity's Ofx. I don't have other OFX so I am not sure about it.

Reply:
I can verify this is also true for vanguard treasury.